astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -873,3 +873,16 @@ def test_empty_quotes(parallel, read_basic):
     table = read_basic('a b\n1 ""\n2 ""', parallel=parallel)
     expected = Table([[1, 2], [0, 0]], names=('a', 'b'))
     assert_table_equal(table, expected)
+
+@pytest.mark.parametrize("parallel", [True, False])
+def test_fast_tab_with_names(parallel):
```

Comment:
This needs to be changed to pass in `read_tab` as an argument, as it is now a fixture--somewhat odd I admit.  Sorry 'bout that.